lib/src/lib/button/models/action-button.model.ts

```diff
@@ -8,7 +8,7 @@ export interface ActionButton<T = any> {
   action?: (data?: T, e?: Event) => any;
   type?: propValue<T, 'toggle' | Lab900ButtonType>;
   color?: propValue<T, ThemePalette>;
-  disabled?: propValue<T, boolean>;
+  disabled?: propValue<T, boolean> | ((data?: T) => propValue<T, boolean>);
```

Comment:
I think `propValue<T, boolean>` already is a mapping of data to a boolean, so i don't thin this change is needed